app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for one_person in people:
+        person_list.append(Person(one_person["name"], one_person["age"]))
+    for i in range(len(people)):
+        if "wife" in people[i] and None is not people[i]["wife"]:
```

Comment:
It would be better to write second condition like this:
```
people[i]["wife"] is not None
```
It would be more readable